scheduler/kubernetes/install.md

```diff
@@ -51,60 +51,37 @@ The native portworx driver in Kubernetes supports the following features:
 
 If you are installing on [Openshift](https://www.openshift.com/), follow [these instructions](/scheduler/kubernetes/openshift-install.html).
 
-PX can be deployed with a single command as a [Kubernetes DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) with the following:
+Portworx gets deployed as a [Kubernetes DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/). Following sections describe how to generate the spec files and apply them.
+
+#### Generating the spec
+
+To generate the spec file, head on to [http://install.portworx.com](http://install.portworx.com) and fill in the parameters.
+
+Alternately, you can use `curl` to generate the spec as described in [Generating Portworx Kubernetes spec using curl](/scheduler/kubernetes/px-k8s-spec-curl.html).
+
+>**Secure ETCD:**<br/> If using secure etcd provide "https" in the URL and make sure all the certificates are in the `/etc/pwx/` directory on each host which is bind mounted inside PX container.
+
+#### Applying the spec
+
+Once you have generated the spec file, deploy Portworx.
 
 ```bash
-# Download the spec - substitute your parameters below
-VER=$(kubectl version --short | awk -Fv '/Server Version: /{print $3}')
-curl -o px-spec.yaml "http://install.portworx.com?c=mycluster&k=etcd://etc.company.net:2379&kbver=$VER"
```

Comment:
I see that this got moved.
Note that the `VER=..` line is pretty important, as this is the only place where we show customers how to correctly collect the Kubernetes version (ie. GKE, or other Kubernetes varieties moving forward), which applies to both "curl" or WebSVC

Reply:
I fixed this in incremental. Add a line and command to make sure they supply kubernetes version even when using the html page.